stats/stats_test.go

```diff
@@ -0,0 +1,273 @@
+package stats
```

Comment:
Really glad you included tests for this. TY!

Reply:
🕺